test/net/sourceforge/kolmafia/request/FightRequestTest.java

```diff
@@ -671,4 +674,22 @@ public void canTrackBellydancerPickpocket() {
     parseCombatData("request/test_fight_bellydancing_pickpocket_3.html");
     assertEquals(3, Preferences.getInteger("_bellydancerPickpockets"));
   }
+
+  @ParameterizedTest
+  @CsvSource({"gain, 2", "lose, -3"})
```

Comment:
`@CsvSource({"gain_2, 2", "lose_3", -3})` would avoid you having to have lose_-3 in your filename...